src/main/java/edu/greenblitz/robotName/subsystems/shooter/Pivot/Pivot.java

```diff
@@ -93,13 +90,18 @@ public double getVelocity() {
 		return pivotInputs.velocity;
 	}
 
-	public double getAngleInRadians() {
-		return pivotInputs.position;
-	}
-
-	public boolean isAtAngle(Rotation2d angle) {
-		return Math.abs(angle.getRadians() - getAngleInRadians()) <= TOLERANCE;
-	}
+    public Rotation2d getAngle() {
+        return Rotation2d.fromRadians(pivotInputs.position);
+    }
 
+    public boolean isAtAngle(Rotation2d angle) {
+        return Math.abs(angle.getRadians() - getAngle().getRadians()) <= TOLERANCE;
+    }
 
+    public Pose3d getPivotPose() {
```

Comment:
![image](https://github.com/GreenBlitz/CRRRRESCENDO/assets/26098399/f7c0dcb5-2451-4953-a1e1-6c70435336dd)
I heard this is a great show, but why do we care about it's pivot?